.github/workflows/cicd.yaml

```diff
@@ -83,6 +83,33 @@ jobs:
       - name: Build website
         run: npm run build
 
+    # Checks that the API reference docs built with shpinx build
+  test-api-reference-docs:
```

Comment:
Moved the previous `sphinx_docs.yaml` here. Now we're also building the docs on every PR.